### DISCUSSION TITLE
Block editor not pre-populated

### DISCUSSION THREAD

**Discussion by michaeljcoyne** [pboivin/twill-form-templates#5] [General]
Hi @pboivin 

I have followed tutorial and it works apart from the prefilled blocks for me. I am wondering if there are any steps missing from docs?

Thanks

**Comment by pboivin** [comment]
Hi @michaeljcoyne, thanks for giving this package a go! I just did a quick test and was able to get the prefilled block selection working. A few things that come to mind, make sure to:

- add the `HandleFormTemplates` trait to your module's repository 
- define a `block_selection` key for the templates that should prefill the block editor
- enable the `block_editor` field in the form for pages that support it. ([I added a note to the README for this](https://github.com/pboivin/twill-form-templates/commit/2722784c3bd9401f7f821213e7928a2ef4f88806))

I hope this can be useful!

**Comment by michaeljcoyne** [comment]
Hi @pboivin I have double checked done all that as per installation. Strange it isnt working for me.

**Comment by michaeljcoyne** [comment]
@pboivin  ok got it working, HandleFormTemplates has to be used at the end of the traits in the repository as another trait was interfering, my guess is the HandleBlocks one. Thanks
